src/Symfony/Component/ErrorCatcher/Exception/FlattenException.php

```diff
@@ -244,16 +244,6 @@ public function getTrace()
         return $this->trace;
     }
 
-    /**
-     * @deprecated since 4.1, use {@see setTraceFromThrowable()} instead.
```

Comment:
@yceruto About this method : isn't it wrong to introduce a new component in 4.4 that has an already deprecated method since 4.1 ? 🤔 

WDYT of keeping this method in the `Debug` `FlattenException` class?

Reply:
That feels also strange to add in the CHANGELOG that we removed a method that was never marked as deprecated in the CHANGELOG before.

Reply:
please remove the changelog entry. having the deprecated method also in the ErrorCatcher makes upgrading easier as you only need to change the class namespace. otherwise you also need to check if all the method are the same/exist.

Reply:
I guess the changelog doesn't hurt